package.json

```diff
@@ -19,34 +19,34 @@
   "main": "src/index.js",
   "repository": "autonomoussoftware/metronome-sdk-status",
   "scripts": {
-    "check": "dependency-check . && dependency-check . --unused --no-dev",
-    "check:outdated": "npm-check -i husky",
```

Comment:
`husky` is now in the latest version so no need to ignore it anymore (it's the one MIT licensed)